source/iterators.tex

```diff
@@ -1110,7 +1110,7 @@
 iterator reference
 \tcode{i}
 by
-\tcode{n}.
```

Comment:
This is still awkward. How about something like:

> The iterator `i` is incremented by $n$ if $n$ is non-negative, and decremented by $-n$ otherwise.

Reply:
I'd like to keep the active voice. I'll propose:
> Increments `i` by `n` if `n` is non-negative, and decrements by `-n` otherwise.

Reply:
Ick. And while I'm here, I should change the Requires:
```diff
-Requires:
+Expects:
 n shall be negative only for bidirectional
-and random access
 iterators.
```

Reply:
Both sound good, but maybe make the second bit a separate change? Both commits can go in the same pull request of course.